### ISSUE TITLE
Missing icon in the top bar on Ubuntu 22.04

### ISSUE THREAD

**Issue by VladMasarik** [gnome-pomodoro/gnome-pomodoro#629]
## Issue Summary

I cannot see the usual "progress circle" in the the top bar. I had no issues with pomodoro on Ubuntu 20.04, but after switching to Ubuntu 22.04, it is no longer showing the "progress circle. Any thoughts? I dont think it is critical, but I like to see the progress.

## Steps to Reproduce

1. Start pomodoro app
2. Start counter
3. Close the app with "x" in the GUI
4. There is no "progress circle" icon in the top right bar
![image](https://user-images.githubusercontent.com/28529900/178270137-a36b8dda-66f9-4728-b971-c9f926de6858.png)

## Context (logs, error output, screencast, OS, Desktop  etc)

Ubuntu 22.04
Gnome 42.2

## Expected Behavior

I would see the progress circle when I am running a timer, and I close the window

## Current Behavior

I dont see the progress circle when running a timer, and I close the window

**Comment by Nihta** [comment]
You can enable it manually using gnome Extension Manager software.

![image](https://user-images.githubusercontent.com/44351310/179940353-729e6abc-2369-4870-89c0-d1fbe95cb03a.png)

**Comment by an-idiot-on-a-branch** [comment]
Unfortunately for some reason I can't see it in the extensions menu

**Comment by VladMasarik** [comment]
@Nihta Lovely! It worked for me.

@an-idiot-on-a-branch I installed the Extension Manager with 
```
$ sudo apt install gnome-shell-extension-manager
```
Then you can either open it with 
```
$ extension-manager
``` 
or just open the Ubuntu search with super(windows) key, and type extension. I had to enable the toggle in the top right corner, although I have no idea what it does... so that would be useful to know, but either way it works thanks to @Nihta 

I guess, I will keep the issue open, for others? We have only a work around for now.

**Comment by dmitriy-chernysh** [comment]
Hi everyone. I had the same issue on Ubuntu 22. But after I turned this extinsion on and "it's time to take a break" comes , the timer  is ticking but screen is freezed. I am not able to cancel it. Mouse and keyboard stops working, just only hard reset helps.

**Comment by benhbell** [comment]
for me hitting the esc key gets me out of that state.

**Comment by Maltergate** [comment]
> Hi everyone. I had the same issue on Ubuntu 22. But after I turned this extinsion on and "it's time to take a break" comes , the timer is ticking but screen is freezed. I am not able to cancel it. When time to break is over, I continue see overlayed timer with the value 0:01. Just hard reset helps.

I do have the same problem with Ubuntu 22.04.01, Gnome 42.2, X11. Without enabling the extension, everything is fine. With the extension enabled, the overlay for taking the break is impossible to get rid of, even by typing escape. The keyboard is somewhat frozen as I can type text but some shortcuts are still working (like F12 to display ddterm, which I can toggle behind the overlay during a break).

**Comment by VladMasarik** [comment]
@Maltergate @dmitriy-chernysh At first nothing worked for me, but as @mpjmuniz  mentioned it is possible.
TL;DR Build it from source according to the website, and reboot https://gnomepomodoro.org/

It seems that these issues have been already discussed and fixed before #613 #619 #622

In the end I realized that I was using version `20.3`, which was installed with `apt` https://packages.ubuntu.com/jammy/gnome-shell-pomodoro . However, this version is rather old, and actually contains the fix for our issue. Thus, I suggest removing the `apt` version first. Then going to the website https://gnomepomodoro.org/ , and installing Pomodoro by building it from source, and rebooting.

I had to enable the notifications in tray again from extensions manager, but afterwards it all worked as expected!

Well, there was a bit a problem when I couldn't leave the "Take a break" overlay, but I think that happened because I pressed the `Super` key at the same time. Locking the screen `Super + L`, and signing in, resolved it.

Also, I don't think it was an issue with Wayland, but I realized that in some cases Wayland is default on Ubuntu 22.04. You can check with
```
loginctl list-sessions
loginctl show-session 2
```
under `type` there will be whether you have wayland or `x11`. In the end I switched to `x11`.

I will leave this issue still open, as the tray icon is not displayed by defualt and you need the extensions manager, but for now the work around is fine.

**Comment by VladMasarik** [comment]
@namphan22 did you try the fix that I explained in the previous comments?

**Comment by kamilprusko** [comment]
If there's no indicator, check Extensions Manager first:
1. make sure extensions are enabled
2. if Pomodoro isn't listed, log out
3. you may need to manually enable the Pomodoro extension

**Comment by namphan22** [comment]
thanks, everyone so much!

**Comment by MikeDoouglas** [comment]
It helped me!

In my case, I just couldn't enable it on Extension Manager so I had to:

1. Build from the source as mention on https://gnomepomodoro.org/
2. Logout from my Ubuntu session
3. Done!

**Comment by mrugeshrocks** [comment]
After reboot or logout enable it from extension manager. This worked for me
